packages/material-ui-lab/src/SpeedDial/SpeedDial.js

```diff
@@ -303,6 +304,11 @@ SpeedDial.propTypes = {
    * @ignore
```

Comment:
Just noticed that some of the props above refer to the Button component, whereas the Fab component was extracted out of the Button. Do you want to fix those up?

Reply:
@mbrookes I'll address the SpeedDialAction color as part of this PR -- I think that makes sense.  For expediency, I'm good with fixing the props as part of the PR also.

Reply:
So... changing from ButtonProps to FabProps is a breaking change... but so is changing the default color for the action buttons but won't break code.  Any ground rules for how breaking changes impact "Lab" components?  I'm now thinking maybe it's best to change the prop name (ButtonProps -> FabProps) in a separate PR

Reply:
Technically the only difference is how we change the version number. However, we are a bit more liberal with breaking changes in the lab since the API isn't considered stable.

Reply:
I'm joining, what's the reasoning against?
```diff
<SpeedDial FabProps={{ color: 'secondary' }} />
```

Reply:
No reasoning against that... it just didn't occur to me to express it that way.  I guess I was kind of thinking the SpeedDial "is a" special type of Fab instead of "has a" Fab (if that makes any sense)... so I expressed it the same way you would with a Fab (as a direct prop).  We're already using `FabProps` (or will be in a subsequent PR) so I'm on board with removing the direct `color` prop and moving it to `ButtonProps`

Reply:
Should I just change `ButtonProps` to `FabProps` as part of this PR instead of a separate PR?